apps/bbsync/tests/test_query.py

```diff
@@ -85,6 +86,51 @@ def test_restore_original(self):
         for key in srtnotes_json.keys():
             assert cf_srtnotes_json[key] == srtnotes_json[key]
 
+    def test_generate_acknowledgments(self):
+        """
+        test generating of SRT acknowledgments attribute array
+        """
+        flaw = FlawFactory()
+        FlawCommentFactory(flaw=flaw)
+        FlawMetaFactory(flaw=flaw)  # not an ack
```

Comment:
Nitpick: I would probably set some actual type. I know that the `FlawMetaFactory` currently produces all the FlawMetas with `REFERENCE` type but we might change this in the future to generate from the allowable choices so I would explicitly state some other `FlawMetaType` here.